src/components/breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2025 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import styled from '@emotion/styled';
+import { forwardRef } from 'react';
+import { truncate } from '~common/helpers/styles';
+import { LinkHighlight } from '../links';
+import { LinkStandalone, LinkStandaloneProps } from '../links/LinkStandalone';
+
+type LinkProps = Omit<LinkStandaloneProps, 'children' | 'isDisabled'> & {
+  linkElement: React.ReactNode;
+};
+
+export type BreadcrumbLinkProps = (
+  | (LinkProps & { isChild?: never })
+  | (Omit<LinkProps, 'to'> & { isChild: true; to?: '' })
+) & {
+  hasEllipsis?: boolean;
+};
+
+const BreadcrumbLinkContainer = styled.span`
+  font-size: var(--echoes-font-size-10);
+  line-height: var(--echoes-line-height-10);
```

Comment:
Use `echoes-typography-text-small-regular` instead. And you probably don't even need it, since you already define it on the BreadcrumbLink